src/cis/fam_cis_client.cpp

```diff
@@ -54,7 +54,8 @@ Fam_CIS_Client::Fam_CIS_Client(const char *name, uint64_t port) {
     /** sending a start signal to server **/
     ::grpc::Status status = stub->signal_start(&ctx, req, &res);
     if (!status.ok()) {
-        throw CIS_Exception(FAM_ERR_RPC, (status.error_message()).c_str());
+        message << "Fam CIS Client: Connect failed : "<<name_s;
```

Comment:
Here we will need to get the output from GRPC too. Probably something like :
status.error_message()).c_str() + ":" + name_s

Reply:
Modified code to address the review comment.  Earlier, when including output from GRPC, there was duplication of 'Connect Failed' as shown below. "Error msg: Fam Memory Service: Connect failed : Connect Failed:127.0.0.1:8795".  Therefore, I had not added grpc error code in the original PR to prevent duplication..
Now, I have modified code in such a way that, OpenFAM does not print the duplicate 'Connect Failed' and only the 'Connect Failed' from grpc is printed.
The net effect in the final error message output remains unchanged as in the following example:
Error msg: Fam Memory Service: Connect Failed:127.0.0.1:8795